src/components/Link/Link.module.scss

```diff
@@ -5,14 +5,9 @@
   display: inline-block;
   color: var(--primary-color);
   font-weight: var(--body-font-weight-bold);
-  text-decoration: none;
+  text-decoration: underline;
```

Comment:
Hmm I see your point, but then we cannot say we are compliant on this. Do we want to give the client this responsibility?

Reply:
Good point. But I think it's better to have this enabled as default than making exceptions based on "this will not be considered pretty" or "the customer probably would not prefer this" because being WCAG compliant doesn't always means pretty UI.

Thinking out loud: I think we could maybe apply a body class with `wcag-optout` and use the preferred (or "old") styling based on the presence of this body class. This is the most simple solution i could come up with.

Reply:
I like the opting option @langemike proposes. But I think we should first discuss with the team if this should be opt-in or opt-out. (And I think we should only use it for UI affecting things such as the underlined link)

Reply:
We've decided today with the whole team that my initial implementation is sufficient enough. If a client wants to remove this underline, they will need to opt for Tier 2, in which we'll be able to remove it.